File: src/entity/bullet/MissileBullet.java
package entity.bullet;

import entity.enemy.Enemy;
import javafx.scene.image.Image;

public class MissileBullet extends Bullet {
	
	public static final int DAMAGE = 3;
	public static final int SPEED = 4;
	public static final String IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile251.png";
	public static final String FLAMES_IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile295.png";
	
	public MissileBullet(double x, double y, Enemy enemy) {
		super(x, y, enemy, SPEED, DAMAGE);
		
		img = new Image(IMAGE_URL);
		flames = new Image(FLAMES_IMAGE_URL);
	}


}	

File: src/entity/bullet/MissileBullet.kt
package entity.bullet

import entity.enemy.Enemy
import javafx.scene.image.Image

class MissileBullet(x: Double, y: Double, enemy: Enemy) : Bullet(x, y, enemy, MissileBullet.SPEED, MissileBullet.DAMAGE) {
    override var img: Image = Image(IMAGE_URL)

    init {
        flames = Image(FLAMES_IMAGE_URL)
    }

    companion object {

        val DAMAGE = 3
        val SPEED = 4.0
        val IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile251.png"
        val FLAMES_IMAGE_URL = "file:img/PNG/Retina/towerDefense_tile295.png"
    }


}	
